### ISSUE TITLE
On issue view when clicking on new time entries link, i get 500 error page 

### ISSUE THREAD

**Issue by csanquer** [nicStuff/redmine_timesheet_extensions#9]
Hi 

I have a new problem on my redmine test environment 

On issue view when clicking on new time entry link (on actions list ), i got a 500 error page.

related log :

```
Processing TimelogController#new (for 82.243.37.107 at 2011-02-01 18:14:31) [GET]
  Parameters: {"issue_id"=>"3", "action"=>"new", "controller"=>"timelog"}

ActiveRecord::MissingAttributeError (missing attribute: hours):
  app/models/time_entry.rb:47:in `after_initialize'
  app/controllers/timelog_controller.rb:109:in `new'
  app/controllers/timelog_controller.rb:109:in `new'

Rendering /home/redmine/redmine_test-www/public/500.html (500 Internal Server Error)
```

My environment

Redmine 1.1.0
Ruby 1.8.7
gem 1.4.1
rails 2.3.5
rake 0.8.7

web server : mongrel_cluster  with reverse proxy apache vhost

**Comment by nicStuff** [comment]
Hi, recently I had some time to do some attempts, and I found that this bug happens only in a production environment (i.e. with passenger, or launching the server with "-e production"). I was able to reproduce the bug, but I had no time to resolve it, sorry.

The fact that the bug happens only in production makes me suppose that this bug won't have a simple solution....

I will keep you up to date as I do something.

Cheers
